nexus/src/app/iam.rs

```diff
@@ -123,6 +123,20 @@ impl super::Nexus {
         Ok(db_silo_user)
     }
 
+    /// Fetch the currently-authenticated Silo user's Silo
+    pub async fn silo_user_fetch_silo(
+        &self,
+        opctx: &OpContext,
+    ) -> LookupResult<db::model::Silo> {
+        let authz_silo = opctx
+            .authn
+            .silo_required()
+            .internal_context("loading current silo")?;
```

Comment:
Given that (I believe) this is used for debugging it might be better to say something like `fetching silo for current user`